src/scripts/main.js

```diff
@@ -1 +1,61 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('click', (e) => {
+    resolve('Second promise was resolved');
+  });
+
+  document.addEventListener('contextmenu', (e) => {
+    resolve('Second promise was resolved');
+  });
```

Comment:
The second promise resolves on both click and contextmenu events. This means it will resolve twice if both events occur, which might not be the intended behavior. Consider using a flag to ensure it resolves only once.